src/ui_fsmenu/multiplayersetupgroup.cc

```diff
@@ -468,7 +468,14 @@ struct MultiPlayerPlayerGroup : public UI::Box {
 						break;
 					}
 				}
-				if (matches_tags) {
+				bool compatible_wc = true;
+				for (const std::string& wc : addme.incompatible_win_conditions) {
+					if (wc == settings_->get_win_condition_script()) {
+						compatible_wc = false;
+						break;
+					}
+				}
+				if (matches_tags && compatible_wc) {
```

Comment:
```suggestion
				if (matches_tags && !addme.incompatible_win_conditions.count(settings_->get_win_condition_script())) {
```

Reply:
@Nordfrees 
thanks for the suggestion. this is the same code as in singleplayer with the only difference of there I used settings_->settings().win_condition_script and here I tried settings_->get_win_condition_script(). 
While they are delivering the same result is there any advantage/disadvantage in the syntax as I think we should harmonize this.

Reply:
The function simply returns the member variable so it doesn't really matter, but a high-level function (`settings_->get_win_condition_script()`) is usually better style than directly accessing low-level variables.